Content.Shared/Weapons/Ranged/Systems/SharedGunSystem.cs

```diff
@@ -128,6 +130,9 @@ private void OnShootRequest(RequestShootEvent msg, EntitySessionEventArgs args)
             return;
         }
 
+        if (_mobState.IsIncapacitated(user.Value))
+            return;
+
```

Comment:
This should be under AttemptShoot so it works for NPCs as well, not just the network event handler.